compiler/ghc/default.nix

```diff
@@ -397,6 +472,7 @@ stdenv.mkDerivation (rec {
       echo "ERROR: Missing file $out/bin/${targetPrefix}ghc-pkg"
       exit 1
     fi
+    ${ lib.optionalString (!useHadrian) ''
```

Comment:
Why is this not needed for Hadrian?

Reply:
It just didn't work because the directory name was not the same.  I have updated it to check for the `.conf` files and using the path variables so it will work.